packages/create-app/src/commands/init.ts

```diff
@@ -0,0 +1,28 @@
+import {execSync} from 'child_process';
+
+import {Command, Flags} from '@oclif/core';
+import path from 'pathe';
+
+export default class InitCommand extends Command {
+  static description = 'Create new GestaltJS app';
+  static flags = {
+    name: Flags.string({
+      char: 'n',
+      description: 'project name',
+    }),
+    path: Flags.string({
+      char: 'p',
+      description: 'project path',
+    }),
+  };
+
+  async run(): Promise<void> {
+    const {args, flags} = await this.parse(InitCommand);
+    const name = flags.name ? flags.name : 'my-gestalt-project';
+    const projectPath = flags.path ? flags.path : this.config.home;
+    const plopfilePath = path.join(__dirname, '../../dist');
+    execSync(
```

Comment:
Similar comment here regarding using `execSync` from `child_process`. It might lead to incompatibility with other OS. I'd instead use [shelljs](https://www.npmjs.com/package/shelljs). @lascenify [built](https://github.com/gestaltjs/gestalt/pull/18/files#diff-961ad2c6f0a6d2522755403d907035dfa53d317163ebd97a5e29a95af2ef6675) an API for that in `@gestaltjs/support` that you might want to bring into this PR once hers has been merged.